.github/workflows/community.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  issues:
+    types: [ opened ]
+  issue_comment:
+    types: [ created ]
+  pull_request:
+    types: [ opened ]
+  pull_request_review:
+    types: [ submitted ]
+  pull_request_review_comment:
+    types: [ created ]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: "<h1>Looks good having you contribute to this project!</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face: If any questions please ask in [Discord](https://discord.com/invite/jZQs6Wu)"
+          pr-message: "<h1>Looks good having you contribute to this project!</h1> Feel free to create a <strong>Pull Request</strong>! Welcome to the community :nerd_face: If any questions please ask in [Discord](https://discord.com/invite/jZQs6Wu)"
+
+  stats:
```

Comment:
Does this need to be named statistics to work?
I could be wrong. :thinking:

Reply:
Great question, I'm not sure since the only error we have rn is the `checkout` step that you mentioned:
![imagem](https://user-images.githubusercontent.com/18630253/93027487-14a74f80-f605-11ea-9b27-81d5083e1ff4.png)

Reply:
I don't think this would cause it to fail, but it would be inconsistent with the rest of our projects. So it is best to be like the other projects https://github.com/EddieJaoudeCommunity/support/blob/main/.github/workflows/community.yml